x/circuit/README.md

```diff
@@ -0,0 +1,126 @@
+# Circuit Breaker - block specific functionality in the state machine
+
+
+## Concepts
+
+Circuit Breaker is a module that is meant to avoid a chain needing to halt/shut down in the presence of a vulnerability, instead the module will allow specific messages or all messages to be disabled. When operating a chain, if it is app specific then a halt of the chain is less detrimental, but if there are applications built on top of the chain then halting is expensive due to the disturbance to applications. 
+
+Circuit Breaker works with the idea that an address or set of addresses have the right to block messages from being executed and/or included in the mempool. The address that disabled the message from being executed also has the right to reset the circuit breaker and allow the application to resume execution of the message. 
```

Comment:
Actually I think any authorized account can reset the circuit breaker. it doesn't need to be the one that tripped it.

Reply:
that works. don't see a specific case for leaving it to only those with perms

Reply:
Yes, I would mention this too

Reply:
adjusted wording.